invenio_app_rdm/theme/views.py

```diff
@@ -75,6 +90,34 @@ def init_menu():
             order=1,
         )
 
+        current_menu.submenu("cog.record").register(
+            "administration.Records",
+            _("Manage record"),
+            order=1,
+            endpoint_arguments_constructor=lambda: {
+                "q": f"id:{request.view_args.get('pid_value')}"
+            },
+            visible_when=_can_manage_record,
+        )
+
+        current_menu.submenu("cog.user").register(
+            "administration.users",
+            _("Manage user"),
+            order=2,
+            endpoint_arguments_constructor=lambda: {"q": f"id:{current_user.id}"},
+            visible_when=_can_manage_record,
+        )
+
+        current_menu.submenu("cog.community").register(
+            "administration.communities",
+            _("Manage community"),
+            order=3,
+            endpoint_arguments_constructor=lambda: {
+                "q": f"slug:{request.view_args.get('pid_value')}"
```

Comment:
Comment for reviewers: In the [issue](https://github.com/zenodo/rdm-project/issues/262) is specified to user UUID, however in the request we do only have access to the slug, to get more infor we would need to resolve the community, if I am not mistaken. Since the slug is unique we should be good IMHO.